lms/services/canvas_api/_pages.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass
+from typing import List
+
+from marshmallow import fields
+
+from lms.validation import RequestsResponseSchema
+
+
+@dataclass
+class CanvasPage:
```

Comment:
This dataclass adds a bit of noise serializing/unserializing it. 

I reckon it's worth the effort, the alternative, `dict typed` interfaces here has proved to be confusing in the files counterpart.